x-pack/plugins/stack_alerts/public/alert_types/es_query/rule_common_expressions/threshold_help_popover.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { Component } from 'react';
+import {
+  EuiButtonIcon,
+  EuiPopover,
+  EuiPopoverTitle,
+  EuiText,
+  EuiCallOut,
+  EuiSpacer,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+
+interface State {
+  isPopoverOpen: boolean;
+}
+
+export class QueryThresholdHelpPopover extends Component<{}, State> {
+  state: State = {
+    isPopoverOpen: false,
+  };
+
+  _togglePopover = () => {
+    this.setState((prevState) => ({
+      isPopoverOpen: !prevState.isPopoverOpen,
+    }));
+  };
+
+  _closePopover = () => {
+    this.setState({
+      isPopoverOpen: false,
+    });
+  };
+
+  _renderContent() {
+    return (
+      <div>
```

Comment:
To keep the popover from expanding the width of the screen, perhaps just adding a max-width will move this forward. 

<img width="700" alt="image" src="https://user-images.githubusercontent.com/3756330/178768102-be3fe03a-20f3-49fe-937c-90415bc5e7ab.png">

Can do it inline here with a style tag, or via emotion (for just one property, maybe overkill, but makes it easier to find in the code)

```
import { css } from '@emotion/react';

const PopoverStyles = css`
max-width: 400px;
`

<div css={PopoverStyles}> ... 


```


I had used 400px in the screenshot.  Arbitrary number, so feel free to adjust if you feel necessary.

Reply:
@mdefazio I having a hard time with the pop-up covering the fields I asked for help on.  Can we change its placement?

Reply:
It might help if we put the icon before the label (at the left side of the UI). I'll give that a try.

Reply:
Thanks for the information!  I've added that css change and to avoid covering up the fields I've changed `anchorPosition` to `upLeft`.